### DISCUSSION TITLE
filters, table and toast components

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#723] [Q&A] [answered]
Hello,

I am working on a design that will use filters, table and toast components. I see table and toast components are soon to be released as a paste library. I am curious will it have filter and sort components as well? I was wondering if I can have early access for to any of those components for design solution that we like to do usability studies for in the month of Sept or October?  If there is still time to release these components, what is my alternative to build the design solution? React components?

Thank you.

**Comment by richbachman** [comment, marked as answer]
Hi @ishah-twilio. Thanks reaching out about the toast and table components. We'll be launching the Toast component toward the end of September, so you should be able to use that. We'll let you know when we files for testing.

For tables and filters, you can continue to use the Console React components for now.

**Reply by ishah-twilio** [reply]
Thank you
